scripts/e2e-tests.sh

```diff
@@ -228,6 +243,10 @@ function create_test_cluster() {
   fi
   local result="$(cat ${TEST_RESULT_FILE})"
   echo "Test result code is $result"
+
+  # Save some metadata about cluster creation for using in prow and testgrid
+  save_metadata
```

Comment:
I think this should be done during setup, so if for any reason the teardown doesn't execute, the metadata is still there.